specs/2.0.0/index.html

```diff
@@ -1567,7 +1567,8 @@ <h4>Texture objects</h4>
         Specify all the levels of a two-dimensional or cube-map texture at the same time. <br><br>
 
         The image contents are set as if a buffer of sufficient size initialized to 0 would be passed to
-        each texImage2D call in the pseudocode in The OpenGL ES 3.0 specification section 3.8.4
+        each texImage2D (or compressedTexImage2D for compressed formats) call in the pseudocode in
+        The OpenGL ES 3.0 specification section 3.8.4
```

Comment:
This change needs to be applied to specs/latest/2.0/index.html as well - could you please upload a pull request for that too? Thank you.